controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java

```diff
@@ -375,6 +375,7 @@ private ReaderGroupConfiguration getRGConfigurationFromRecord(final String scope
                                          .thenCompose(x -> eventHelper.checkDone(() -> isRGCreated(scope, rgName, context))
                                          .thenCompose(done -> buildCreateSuccessResponse(scope, rgName, context)))));
                      }
+                     log.info("Reader Group {} already exist", rgName);
```

Comment:
```suggestion
                     log.info("Reader Group {} already exists", rgName);
```

Reply:
Please print scoped readergroup name here. You can use utility methods from NameUtils.java to build a scoped ReaderGroup name.

Reply:
Please print scoped readergroup name here. You can use utility methods from NameUtils.java to build a scoped ReaderGroup name.

Reply:
Changed log message as per suggestion.